server/app.js

```diff
@@ -312,6 +340,35 @@ app.get('/user/:username', async (req, res, next)=>{
 	return next();
 });
 
+//Rename Brews
```

Comment:
Can this be something like "Change author name on brews" or something? We are not renaming the brews.

Reply:
true that